packages/help/src/Help.tsx

```diff
@@ -0,0 +1,32 @@
+//@ts-nocheck
```

Comment:
Why do you need this exception?

Reply:
Because popover component isn't yet migrated into TS. So ts compiler need a declaration file for this dependency. All ts-nocheck or ts-ignore must be remove when all packages will be migrated.